.github/workflows/deploy.yml

```diff
@@ -115,7 +115,7 @@ jobs:
           REDIS_PORT: 6379
           REDIS_DB: 0
           METADATA_BUCKET: hathor-explorer-metadata-testnet
-          CORS_ALLOWED_REGEX: https?://([a-z0-9]*\.){0,5}hathor\.network
+          CORS_ALLOWED_REGEX: https?:\/\/([a-z0-9]*\.){0,5}hathor\.network|https?:\/\/([a-z0-9\-]*\.?){0,5}hathor56\.wpcomstaging\.com
```

Comment:
question: Can we have an example of the regex this is supposed to match?

From what i'm readind it can match up to 5 sub domains then `hathor56.wpcomstaging.com`.
So `https://foo.bar.hathor56.wpcomstaging.com` would be valid.

Reply:
You right! I left it open on purpose, but we can set a stricter value. We need to enable these two URLs:
- https://staging-94c0-hathor56.wpcomstaging.com
- https://hathor56.wpcomstaging.com/

Reply:
Maybe we can use this regex: `https?:\/\/(staging-94c0-)?hathor56\.wpcomstaging\.com`. What do you think?

Reply:
Great. didn't catch that you added an optional dot so we can use `...-hathor56...` 
But I think we can leave it open, make it easier to add more domains, as long as